chat-ui/src/DevCenterChatbot.tsx

```diff
@@ -526,26 +564,70 @@ const Message = ({
           : messageData.content}
       </LGMessage>
       {displaySuggestedPrompts && (
-        <div className={styles.message_prompts}>
-          <MessagePrompts label="Suggested Prompts">
-            {suggestedPrompts.map((sp, idx) => (
+        <MessagePrompts
+          messagePrompts={suggestedPrompts}
+          messagePromptsOnClick={suggestedPromptOnClick}
+        />
+      )}
+    </Fragment>
+  );
+};
+
+type MessagePromptsProps = {
+  messagePrompts: string[];
+  messagePromptsOnClick: (prompt: string) => void;
+};
+
+const MessagePrompts = ({
+  messagePrompts,
+  messagePromptsOnClick,
+}: MessagePromptsProps) => {
+  const [inProp, setInProp] = useState(true);
+  const [suggestedPromptIdx, setSuggestedPromptIdx] = useState(-1);
+  const nodeRef = useRef(null);
+
+  const duration = 300;
+
+  const defaultStyle = {
+    transition: `opacity ${duration}ms ease-in`,
+    opacity: 0,
+  };
+
+  const transitionStyles = {
+    entering: { opacity: 1 },
+    entered: { opacity: 1 },
+    exiting: { opacity: 0 },
+    exited: { opacity: 0 },
+    unmounted: { opacity: 0 },
+  };
+
+  return (
+    <Transition in={inProp} timeout={duration} nodeRef={nodeRef}>
+      {(state) => (
+        <div
+          className={styles.message_prompts}
+          style={{ ...defaultStyle, ...transitionStyles[state] }}
+        >
```

Comment:
Per my earlier comment about using emotion & CSSTransition. Also make sure to import `CSSTransition` too!

```suggestion
    <CSSTransition
      in={inProp}
      timeout={duration}
      nodeRef={nodeRef}
      classNames={styles.message_prompts}
    >
      <div
        className={styles.message_prompts}
        ref={nodeRef}
      >
```